common-tooling/action.yml

```diff
@@ -262,6 +264,32 @@ runs:
       cache-dependency-path: ${{ inputs.python-cache-dependency-path }}
       update-environment: ${{ inputs.python-update-environment }}
       allow-prereleases: ${{ inputs.python-allow-prereleases }}
+
+  - name: Install Python3 (ubuntu-arm install)
+    uses: deadsnakes/action@v3.1.0
+    # actions/setup-python doesn't yet support ARM, see https://github.com/actions/setup-python/issues/705#issuecomment-1756948951
+    if: ${{ contains(runner.arch, 'arm') && contains(runner.name, 'ubuntu') && inputs.python-enabled == 'true' && ( env.python_missing == 'true' || inputs.overwrite == 'true' ) }}
+    with:
+      python-version: ${{ inputs.python-version }}
+      python-version-file: ${{ inputs.python-version-file }}
+      cache: ${{ inputs.python-cache }}
+      architecture: ${{ inputs.python-architecture }}
+      check-latest: ${{ inputs.python-check-latest }}
+      token: ${{ inputs.python-token }}
+      cache-dependency-path: ${{ inputs.python-cache-dependency-path }}
+      update-environment: ${{ inputs.python-update-environment }}
+      allow-prereleases: ${{ inputs.python-allow-prereleases }}
```

Comment:
the only inputs supported by `deadsnakes/action` are
- python-version
- debug
- nogil